pkg/apis/operator/v1alpha1/tektonconfig_types.go

```diff
@@ -186,4 +186,6 @@ type Platforms struct {
 	// OpenShift allows configuring openshift specific components and configurations
 	// +optional
 	OpenShift OpenShift `json:"openshift,omitempty"`
+	// +optional
+	Kubernetes Kubernetes `json:"kubernetes,omitempty"`
```

Comment:
Can't we manage PAC outside of the platform, as it will be supported on both OpenShift and Kubernetes, just like other components such as Results and Chains?

Reply:
We could, but the original design specifically introduced spec.platforms.openshift.pipelinesAsCode for PAC and even named the CRD as openshiftpipelinesascodes.operator.tekton.dev. I agree with your point, but since this structure is already in place, we’ll unfortunately need to stick with this platform setup to maintain backward compatibility for PAC on OpenShift clusters imho. Unforunately this also means that the crd and the code references will also have openshiftpiplelinesascode instead of pipelinesascode. We could move it to just PAC at a later stage I guess.

Reply:
I would defer to @vdemeester and @jkhelil on that. I core issue here is the the crd and code was defined to be specific to ocp. If we were to switch to pitpelinesascodes.operator.tekton.dev obviosuly upgrade breaks andbackwards compatibility is lost. I think we have have a follow up PR which has pipelinesascodes.operator.tekton.dev cr along with openshiftpipelinesascodes.operator.tekton.dev and after a depreatiation period remove openshiftpipelinesascodes.operator.tekton.dev completely. How does that sound?